test/jasmine/tests/histogram_test.js

```diff
@@ -162,10 +163,20 @@ describe('Test histogram', function() {
 
 
     describe('calc', function() {
-        function _calc(opts) {
-            var base = { type: 'histogram' },
-                trace = Lib.extendFlat({}, base, opts),
-                gd = { data: [trace] };
+        beforeAll(function() {
+            jasmine.addMatchers(customMatchers);
+        });
+
+        function _calc(opts, extraTraces) {
```

Comment:
modified test images in https://github.com/plotly/plotly.js/pull/1944/commits/3f173a447fcb09096026230501e0662916af8971 - previously trace1 would have gotten bin size 2, so bin centers at 1 & 2 for trace0, 0.5, 2.5, and 4.5 for trace1 and displayed bar widths of 0.5. Now they all get bin size = displayed bar width = 1